pkg/airgap/update.go

```diff
@@ -146,6 +146,7 @@ func UpdateAppFromPath(a *apptypes.App, airgapRoot string, airgapBundlePath stri
 	}
 
 	pullOptions := pull.PullOptions{
+		Downstreams:            []string{"this-cluster"},
```

Comment:
since this is an update, not an initial install, why not read the downstream names from the database like the online path does? https://github.com/replicatedhq/kots/blob/b5e4e5b2db303c3b5c258891901542498c492e09/pkg/kotsadmupstream/upstream.go#L159